### PR TITLE
[SUPPORT] migration of Caching content

### PR DESCRIPTION
**Pull request by jedecf**
PCX-6635

### CODE REVIEW

File: content/cache/troubleshooting/_index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+pcx_content_type: navigation
+title: Troubleshooting Caching
```
Comment (angelampcosta) [inline content/cache/troubleshooting/_index.md]:
```suggestion
title: Troubleshooting
```

File: content/cache/troubleshooting/_index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+pcx_content_type: navigation
+title: Troubleshooting
+weight: 5
+---
+
+# Troubleshooting Caching
```
Comment (angelampcosta) [inline content/cache/troubleshooting/_index.md]:
```suggestion
# Troubleshooting
```

File: content/cache/troubleshooting/_index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (angelampcosta) [inline content/cache/troubleshooting/_index.md:10]:
```suggestion

The following topics are useful for troubleshooting Cache issues.

```

**Comment by kodster28** [comment]
@jedecf, can you change the filenames to match how they're rendered in the URLs?

i.e., `Best Practices Speed up your Site with Custom Caching via Cloudflare Page Rules.md --> best-pratices-speed-up-your-site-with-custom-caching-via-cloudflare-page-rules.md`

That prevents the spaces from breaking our breadcrumbs + makes them align with our standards.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**